packages/namekit-react/src/components/DisplayedPrice.tsx

```diff
@@ -0,0 +1,213 @@
+import {
+  Currency,
+  type Price,
+  formattedPrice,
+  PriceCurrencyFormat,
+} from "@namehash/ens-utils";
+
+import React from "react";
+import {
+  PriceSymbol,
+  PriceSymbolPosition,
+  PriceDisplayPosition,
+  PriceDisplaySize,
+  PriceSymbology,
+} from "./PriceSymbol";
+
+import cc from "classcat";
+import { Tooltip } from "@namehash/nameguard-react";
+
+interface PriceProps {
+  // The price to be displayed
+  price: Price;
+  // The alternative price to be displayed (the price converted to another currency)
+  altPrice?: Price;
+  // The onClick event handler for the price text
+  onTextClick?: () => void;
+  // Wether to display the alternative price as a text instead of a tooltip
+  altPriceDisplayedAsText: boolean;
```

Comment:
Suggest converting this from a boolean into an enum where each enum value is more self documenting and descriptive.